src/App.tsx

```diff
@@ -1,18 +1,21 @@
-import logo from "./logo.svg";
 import "./App.css";
 import Header from "./components/Header/Header";
 import Carousel from "./components/Carousel/Carousel";
 import Filter from "./components/Filter/Filter";
 import CardList from "./components/CardList/CardList";
-import Footer from "./components/Footer/Footer";
+import React from "react";
+const Footer = React.lazy(() => import("./components/Footer/Footer"));
```

Comment:
why are we lazy loading Footer?

Reply:
It was just to practice code splitting and see the difference.